tests/common/test_event.py

```diff
@@ -270,107 +298,81 @@ def test_periodic_emits_after_elapsed_delta(self, mock_event):
-        log_msg = "{0}\n{1}".format("DummyCmd", "\n".join([line for line in msg.split('\n') if line != ""]))
-
-        with patch("azurelinuxagent.common.event.datetime") as patch_datetime:
-            patch_datetime.utcnow = Mock(return_value=datetime.strptime("2019-01-01 01:30:00",
-                                                                        '%Y-%m-%d %H:%M:%S'))
-            with patch('os.getpid', return_value=42):
-                with patch("threading.Thread.getName", return_value="HelloWorldTask"):
-                    add_event('test_extension', message=log_msg, duration=duration)
-
-        for tld_file in os.listdir(self.tmp_dir):
-            event_str = MonitorHandler.collect_event(os.path.join(self.tmp_dir, tld_file))
-            event_json = json.loads(event_str)
-
-            self.assertEqual(len(event_json["parameters"]), 15)
-
-            # Checking the contents passed above, and also validating the default values that were passed in.
-            for i in event_json["parameters"]:
-                if i["name"] == "Name":
-                    self.assertEqual(i["value"], "test_extension")
-                elif i["name"] == "Message":
-                    self.assertEqual(i["value"], log_msg)
-                elif i["name"] == "Version":
-                    self.assertEqual(i["value"], str(CURRENT_VERSION))
-                elif i['name'] == 'IsInternal':
-                    self.assertEqual(i['value'], False)
-                elif i['name'] == 'Operation':
-                    self.assertEqual(i['value'], 'Unknown')
-                elif i['name'] == 'OperationSuccess':
-                    self.assertEqual(i['value'], True)
-                elif i['name'] == 'Duration':
-                    self.assertEqual(i['value'], 0)
-                elif i['name'] == 'ExtensionType':
-                    self.assertEqual(i['value'], '')
-                elif i['name'] == 'ContainerId':
-                    self.assertEqual(i['value'], 'UNINITIALIZED')
-                elif i['name'] == 'OpcodeName':
-                    self.assertEqual(i['value'], '2019-01-01 01:30:00')
-                elif i['name'] == 'EventTid':
-                    self.assertEqual(i['value'], threading.current_thread().ident)
-                elif i['name'] == 'EventPid':
-                    self.assertEqual(i['value'], 42)
-                elif i['name'] == 'TaskName':
-                    self.assertEqual(i['value'], 'HelloWorldTask')
-                elif i['name'] == 'KeywordName':
-                    self.assertEqual(i['value'], '')
-                elif i['name'] == 'GAVersion':
-                    self.assertEqual(i['value'], str(CURRENT_AGENT))
-                else:
-                    self.assertFalse(True, "Contains a field outside the defaults expected. Field Name: {0}".
-                                     format(i['name']))
-
-    def test_save_event_message_with_decode_errors(self):
```

Comment:
included as a test case in test_collect_events_should_ignore_invalid_event_files